gravity-forms/gw-update-posts.php

```diff
@@ -209,6 +213,11 @@ public function prepare_meta_input( $meta, $post_id, $entry, $form, $meta_input
 			if ( ! rgblank( $meta_value ) ) {
 				$acf_field = $this->acf_get_field_object_by_name( $key, $group );
 				if ( $acf_field ) {
+					$meta_value = $acf_field['type'] == 'google_map' && $field_type == 'address' ? array(
+						'address' => $meta_value,
+						'lat'     => $entry[ "gpaa_lat_{$field->id}" ],
+						'lng'     => $entry[ "gpaa_lng_{$field->id}" ],
```

Comment:
I'd use `rgar` here that way if this meta is missing for some reason, it won't result in a notice/warning.